run_test fails

Several algorithms didn't work properly 

didn't learn ssh-rsa3072-falcon512 correctly
learn additional hostkeys, type=ssh-rsa3072-dilithium2aes
learn hostkeys ssh-rsa3072-dilithium2aes (got 1 wanted 30)
didn't learn ssh-rsa3072-dilithium2aes correctly
learn additional hostkeys, type=ssh-rsa3072-picnicL1full
learn hostkeys ssh-rsa3072-picnicL1full (got 1 wanted 30)
didn't learn ssh-rsa3072-picnicL1full correctly
learn additional hostkeys, type=ssh-rsa3072-sphincsharaka128fsimple
learn hostkeys ssh-rsa3072-sphincsharaka128fsimple (got 1 wanted 30)
didn't learn ssh-rsa3072-sphincsharaka128fsimple correctly
learn additional hostkeys, type=ssh-ecdsa-nistp256-falcon512
learn hostkeys ssh-ecdsa-nistp256-falcon512 (got 1 wanted 30)
didn't learn ssh-ecdsa-nistp256-falcon512 correctly
learn additional hostkeys, type=ssh-ecdsa-nistp521-falcon1024
learn hostkeys ssh-ecdsa-nistp521-falcon1024 (got 1 wanted 30)
didn't learn ssh-ecdsa-nistp521-falcon1024 correctly
learn additional hostkeys, type=ssh-ecdsa-nistp384-dilithium3
learn hostkeys ssh-ecdsa-nistp384-dilithium3 (got 1 wanted 30)
.
.
.
Another issue at the end of executing this command, see below

old key present
didn't learn changed key
check rotate primary hostkey
ssh -oStrictHostKeyChecking=yes -oHostKeyAlgorithms=ssh-ed25519 **failed**
didn't learn changed key
failed hostkey rotate
make[1]: *** [Makefile:224: t-exec] **Error 1**
make[1]: Leaving directory '/home/yaser/openssh/regress'
make: *** [Makefile:730: t-exec] **Error 2**

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -38,8 +38,8 @@ jobs:
           name: Build OpenSSH
           command: env WITH_OPENSSL=<< parameters.WITH_OPENSSL >> ./oqs-scripts/build_openssh.sh
       - run:
-          name: Run short tests
-          command: make tests LTESTS=""
+          name: Run tests documented to pass
+          command: ./oqs-test/run_tests.sh
       - run:
           name: Ensure we have the ssh and sshd syntax right once for each algorithm
           command: python3 oqs-test/try_connection.py doone

--- a/oqs-test/run_tests.sh
+++ b/oqs-test/run_tests.sh
@@ -10,7 +10,9 @@ INVESTIGATE_FURTHER="integrity \
                      keys-command \
                      hostkey-agent \
                      authinfo \
-                     principals-command"
+                     principals-command \
+                     knownhosts-command \
+                     hostkey-rotate"
 SKIPPED_DUE_TO_CERTIFIED_KEYS="agent \
                                cert-hostkey \
                                cert-userkey \

